first round edits without pictures (#500)
PECE-project/drupal-pece: artifact-new.md

You can add many types of artifacts, including audio files, video, images, pdf, text, and websites. Within the "Archive" section of content types, you will see all the artifact options.

Next, click the type of artifact you want to create.

![](media/PECE-new-artifact-screenshot-image.png)

You should now have access to all of the fields you need to fill out in order to create a new image artifact. First, type the title of your image in the “Title” pane.

Next, upload your artifact by clicking the "Choose file" button and selecting the name of the file you would like to upload. 

![](media/PECE-new-artifact-title-screenshot.png)

Click “Open” to upload the file into PECE. Depending on the size of the file it may take a few seconds or minutes to upload; a small icon for the file will appear once uploading is complete.
Add Alt Text as desired.

> Next, scroll down to the metadata fields. The menu bar on the left side will automatically be open to the "Permission, Authorship, and Groups" section. Your screen name should already appear in the first contributor field. To add the names of any other project members who you would like to grant access to editing the artifact, type the first few letters of their name into the other “contributor” field. To add more than one additional contributor, click the "add another item" button and enter their name in the new field.

Reviewer: ```suggestion
Next, scroll down to the metadata fields. The menu bar on the left side will automatically be open to the "Permission, Authorship, and Groups" section. Your screen name should already appear in the first contributor field. Listed contributors have the ability to edit the content, and are also credited as content contributors when the content is published. To add the names of any other project members who you would like to grant access to editing the artifact, type the first few letters of their name into the other “contributor” field. To add more than one additional contributor, click the "add another item" button and enter their name in the new field.
```

> Next, scroll down to the "View Permission and Groups" section. The button switch will automatically set viewing permissions to "Everyone can view this content," making the artifact visible to all internet users. To change viewing access, click the button switch. A new field "People with view access" will pop up. To allow specific people viewing access, type the first few letters of their name and select their username.

Reviewer: ```suggestion
Next, scroll down to the "View Permission and Groups" section. The button switch will automatically set viewing permissions to "Everyone can view this content," making the artifact visible to all internet users. If you want to restrict access to the content, click the button switch to turn this off.

If you choose to leave the toggle on, you can still associate the content with a particular group. To do so, complete the "Group association and view access." You can select a group with which to associate the content by typing its name into the field.

If you choose to restrict access to the content, you can designate specific people or groups that will have view access. To allow specific people viewing access, in the field, "People with view access" type the first few letters of their name and select their username. Add groups whose members should have view access to this memo by typing the group name into the "Group association and view access" field. **Note that if you give a group view access, it will also associate the content with that group. This means the content will appear on the group's page.**
```

> You will also see another field: "Group association and view access." This field indicates groups associated with this content. Selecting a group here will display this content on the group's page. Note that, if you select a group here, all group members will also have the ability to view this content.

Reviewer: ```suggestion
```

Reviewer: replaced with content above to maintain consistency with text

![](media/PECE-new-artifact-permissions.png)

Next, click on the “Source Metadata" section on the left side menu bar. Fill out the section, making sure to follow any project- specific instructions. You may also include the format, language, or location associated with the image if you feel as though they are important or interesting to the meaning or usefulness of the image. Otherwise you may choose to leave them blank.

![](media/PECE-new-artifact-source-metadata.png)

Click Save.

> Next, fill out the "Contribution Metadada" section, making sure to select the correct license and add any critical commentary. For the license, we encourage you to leave it as Attribution, Share Alike CC BY-SA, but you can choose any of the other Creative Commons license as you see fit. Please refer to the Licensing section for more information.

Reviewer: Can you add the link to the license options from the help text here?

> You can enter in the fieldsite for the artifact, or click the button switch to use the location entered in the "Source Metadata" section.

Reviewer: ```suggestion
If you wish to associate this artifact with a fieldsite that you have already created in the platform, you can do so by typing the Title for that fieldsite into the "Fieldsite" field. 
```

Tag the content. As you type, suggested tags will be automatically generated if they exist already in the system. We recommend following theses suggestions and using existing tags if they are not too different from what you were intending, but feel free to make up new tags!

![](media/PECE-new-artifact-contribution-metadata.png)

Next, fill out the "Custom Citation" section, using your platform's preferred citation format.

> Next, you can fill out the "File format" section to indicate the file format of the artifact.

Reviewer: ```suggestion
```

Reviewer: This field should go away soon!

> Finally, scroll back up to the top of the page and click “Save” to save your work. And remember: all of these settings can be revised at time: private artifacts can be made public later, and vice versa, for example.

Reviewer: ```suggestion
Finally, scroll back up to the top of the page and click “Save” to save your work. And remember: all of these settings can be revised at a later time: private artifacts can be made public later, and vice versa, for example.
```
